picard/ui/widgets/scripttextedit.py

```diff
@@ -90,6 +90,11 @@ def highlightBlock(self, text):
             if (next_index > 0) and text[next_index - 1] == '\\':
                 next_index += 1
 
+            # Reached end of text?
+            if next_index >= len(text):
+                self.setFormat(index, next_index - index, self.noop_fmt)
+                break
+
             if (next_index > -1) and text[next_index] == '(':
```

Comment:
I noticed there are plenty of useless parenthesis in this code.

Reply:
Indeed, I cleaned it up in https://github.com/metabrainz/picard/pull/1601/commits/c39c28f2e109cfa7245f0e6612d7ed52468e2d69